rnd/nextgenautogpt/manager/manager.py

```diff
@@ -0,0 +1,33 @@
+import multiprocessing as mp
+import time
+
+import nextgenautogpt.executor.executor as mod_executor
+
+
+def run_manager(server_to_manager: mp.Queue, manager_to_server: mp.Queue) -> None:
```

Comment:
class field member instead of params? 
My proposal is using queues: dict[str, Queue] # with its key as event name though

Reply:
The intent of this PR is to give a program sketch so we can all work together on it.